nixos/modules/services/x11/colord.nix

```diff
@@ -18,22 +18,23 @@ in {
 
   config = mkIf cfg.enable {
 
+    environment.systemPackages = [ pkgs.colord ];
+
     services.dbus.packages = [ pkgs.colord ];
 
     services.udev.packages = [ pkgs.colord ];
 
-    environment.systemPackages = [ pkgs.colord ];
+    systemd.packages = [ pkgs.colord ];
```

Comment:
One disadvantage of using this instead of declaring it in NixOS is that you won't be able to customize the service with NixOS anymore. That might be okay for this service, not sure

Reply:
But can't we still make overrides with `systemd.services.colord` that will have a higher priority over anything in the original unit file?

Reply:
Hmm I actually never thought about that, I don't think it would work, but I never tried it.

Reply:
I have before to solve certain issues (certain environment variables needed to be set for a service).
It's also used to override the upstream services https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/system/boot/systemd.nix#L912.

But yeah it could be limited in certain ways (maybe worth seeing how).

Reply:
Ah I see, seems to work well, disregard my comment then :)

Reply:
> Ah I see, seems to work well, disregard my comment then :)

No problem, thanks for raising awareness.